HW1/task1.py

```diff
@@ -0,0 +1,22 @@
+def check_triangle(a, b, c):
+    # Проверка условия существования треугольника
+    if a >= b + c or b >= a + c or c >= a + b:
+        return "Треугольник с такими сторонами не существует."
+
+    # Проверка сравнения длины каждой стороны с суммой двух других
+    if a > b + c or b > a + c or c > a + b:
+        return "Треугольник с такими сторонами не существует."
+
+    # Определение типа треугольника
+    if a == b == c:
+        return "Треугольник является равносторонним."
+    elif a == b or a == c or b == c:
+        return "Треугольник является равнобедренным."
+    else:
+        return "Треугольник является разносторонним."
```

Comment:
Зная и понимаю, что `return` завершает работу функции, ваш код можно значительно упростить
```suggestion
    if a < b + c and b < a + c and c < a + b:
        # Определение типа треугольника
        if a == b == c:
            return "Треугольник является равносторонним."
        if a == b or a == c or b == c:
            return "Треугольник является равнобедренным."

        return "Треугольник является разносторонним."

    return "Треугольник с такими сторонами не существует."
```